### PR TITLE
[zh] Update network-policies.md

### PR DESCRIPTION
**Pull request by YunWZ**

### CODE REVIEW

**Comment by windsonsea** [comment]
Please squash your commits before merge.

**Comment by YunWZ** [comment]
> Please squash your commits before merge.

done.

File: content/zh-cn/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -2,12 +2,21 @@
 title: 网络策略
 content_type: concept
 weight: 50
+description: >-
+  如果您想在 IP 地址或端口级别（OSI 第 3 层或第 4 层）控制流量，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/network-policies.md:6]:
```suggestion
  如果你想在 IP 地址或端口级别（OSI 第 3 层或第 4 层）控制流量，
```

File: content/zh-cn/docs/concepts/services-networking/network-policies.md [hunk 2]
```diff
@@ -2,12 +2,21 @@
...
+  NetworkPolicy 允许您为集群内以及 Pod 和外部世界之间的流量指定规则。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/network-policies.md:7]:
```suggestion
  NetworkPolicy 允许你为集群内以及 Pod 和外部世界之间的流量指定规则。 
```

File: content/zh-cn/docs/concepts/services-networking/network-policies.md [hunk 3]
```diff
@@ -2,12 +2,21 @@
 title: 网络策略
 content_type: concept
 weight: 50
+description: >-
+  如果您想在 IP 地址或端口级别（OSI 第 3 层或第 4 层）控制流量，
+  NetworkPolicy 允许您为集群内以及 Pod 和外部世界之间的流量指定规则。 
+  您的集群必须使用支持 NetworkPolicy 实施的网络插件。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/network-policies.md:8]:
```suggestion
  你的集群必须使用支持 NetworkPolicy 实施的网络插件。
```

**Review by windsonsea — COMMENTED** [review]
We prefer to use `你` rather than `您`, refer to:

https://kubernetes.io/zh-cn/docs/contribute/localization_zh/

**Comment by Sea-n** [comment]
Hey @YunWZ,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by Sea-n** [comment]
Hey @YunWZ, should we close this PR?

**Pull request closed without merging**